libs-haskell/bazel-runfiles/src/DA/Bazel/Runfiles.hs

```diff
@@ -6,33 +6,70 @@
 -- it is simpler to have all code located here.
 module DA.Bazel.Runfiles
   ( locateRunfiles
+  , locateRunfilesOrThrowError
   , mainWorkspace
   ) where
 
 import Control.Monad.Trans.Maybe
 import Data.Foldable
+import Data.List
+import Data.List.Split (splitOn)
 import System.Directory
 import System.Environment
 import System.FilePath
 
 mainWorkspace :: String
 mainWorkspace = "com_github_digital_asset_daml"
 
-locateRunfiles :: FilePath -> IO FilePath
+locateRunfilesOrThrowError :: FilePath -> IO FilePath
```

Comment:
How about calling this `locateRunfiles` and the other one `locateRunfilesMb` given that `locateRunfiles` seems to be more common?

Reply:
Makes sense, done. I also extended Windows CI build to cover whole `//ledger/sandbox/...` as it works now on Windows.